.ps-rule/dep-suppress.Rule.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
What's that for?

Reply:
Just comes from the template as described in https://microsoft.github.io/PSRule/v2/concepts/PSRule/en-US/about_PSRule_SuppressionGroups/#defining-suppression-groups